references/cli/adopt.go

```diff
@@ -404,11 +558,21 @@ var (
 		the command and make your own assemble rules for the adoption application. You can
 		refer to https://github.com/kubevela/kubevela/blob/master/references/cli/adopt-templates/default.cue
 		to see the default implementation of adoption rules.
+
+		If you want to adopt all resources with resource topology rule to Applications,
+		you can use: 'vela adopt --all'. The resource topology rule can be customized by
+		'--resource-topology-rule' flag.
```

Comment:
does the resource topology use the same rule file? if not, maybe we can call this one to be `relation-rule`

Reply:
Currently the rule is not the same, but we're gonna replace it with the same rule in the future.